plonetheme/onegov/tests/test_navigation_portlet.py

```diff
@@ -223,3 +224,27 @@ def test_default_pages_are_not_listed(self, browser):
         self.assertEquals('Homepage', portlet().css('.parent').first.text)
         self.assertEquals('Other page', portlet().css('.current').first.text)
         self.assertEquals([], portlet().css('.sibling').text)
+
+    @browsing
+    def test_renders_when_parent_not_accessible(self, browser):
+        wftool = getToolByName(self.portal, 'portal_workflow')
+        wftool.setChainForPortalTypes(['Folder'], 'simple_publication_workflow')
+
+        create(Builder('navigation portlet').having(currentFolderOnly=False))
+        parent = create(Builder('folder').titled('Parent'))
+        create(Builder('folder').titled('Top Sibling')
+               .within(parent).in_state('published')).reindexObject()
+        child = create(Builder('folder').titled('Child')
+                       .within(parent).in_state('published'))
+        sibling = create(Builder('folder').titled('Bottom Sibling')
+                         .within(parent).in_state('published'))
+        sibling.reindexObject()
```

Comment:
It didn't work without somehow.
The builder does a ``obj.reindexObject(idxs=['review_state'])``, but it doesn't work properly. I'm not sure why.